pkgs/development/compilers/mozart/default.nix

```diff
@@ -1,91 +1,31 @@
-{ lib
-, fetchFromGitHub
-, fetchurl
-, cmake
+{ stdenv
+, lib
+, callPackage
 , unzip
-, makeWrapper
-, boost169
-, pinnedBoost ? boost169
-, llvmPackages
-, llvmPackages_5
-, gmp
 , emacs
-, jre_headless
-, tcl
-, tk
+, makeWrapper
+, installDesktop ? false
```

Comment:
I don't think we need a flag for this. Pretty sure there's no downside to having desktop support by default

Reply:
The downsize is that the desktop file is non functional for mozart with emacs-nox (a tty is not provided when an application is launched via desktop file). I think it is not a good idea to provide a non working desktop file (the same is true for mozart2-unwrapped).